### PR TITLE
add docs to all models that need it

### PR DESCRIPTION
**Pull request by dave-connors-3**
This is a:
- [x] bug fix PR with no breaking changes
- [ ] new functionality

## Description & motivation
this adds docs to all the models from our pacakge that didn't have them

I also 
- discovered that a couple staging and fct models were also missing tests! added those
- saw that i had an extra line left over from correcting an earlier merge conflict, sneaking that in there too

## Integration Test Screenshot

image

## Checklist
- [ ] I have verified that these changes work locally on the following warehouses (Note: it's okay if you do not have access to all warehouses, this helps us understand what has been covered)
    - [ ] BigQuery
    - [x] Postgres
    - [ ] Redshift
    - [ ] Snowflake
- [ ] I have updated the README.md (if applicable)
- [x] I have added tests & descriptions to my models (and macros if applicable)
- [ ] I have added an entry to CHANGELOG.md

### CODE REVIEW

File: integration_tests/models/dbt_project_evaluator_schema_tests/graph.yml [hunk 1]
```diff
@@ -72,3 +72,21 @@ models:
         tests:
           - unique
           - not_null
+
+  - name: stg_naming_convention_folders
+    description: "This table parses the folder name variables into a model to join into help define structure tests"
```
Comment (b-per) [inline integration_tests/models/dbt_project_evaluator_schema_tests/graph.yml:77]: Just a tiny suggestion for `stg_naming_convention_folders` and `stg_naming_convention_prefixes`.
Could we add in the docs that if someone wants to:
- change the the folder names or model prefixes, they would have to modify the `vars` in `dbt_project.yml`
- and if they have a different structure (e.g. a report or base folder and naming convention) they would need to
  1.  create new vars in `stg_naming_convention_prefixes` 
  2. create a copy of the `stg_` models locally and modify them to their needs
  3. and deactivate the equivalent `stg_` models from the package in `dbt_project.yml`
  
  Writing that makes me realize it is not a super easy flow 😄

**Review by b-per — CHANGES_REQUESTED** [review]
Thanks for that @dave-connors-3 !
I just have a small suggestion

File: integration_tests/models/dbt_project_evaluator_schema_tests/graph.yml (hunk 1 above)
Comment (dave-connors-3) [reply integration_tests/models/dbt_project_evaluator_schema_tests/graph.yml:77]: 😅 yeah i think that is worth documenting -- do you think that is in scope for the project variables section you're writing? #102

Comment (graciegoheen) [reply integration_tests/models/dbt_project_evaluator_schema_tests/graph.yml:77]: ^capture in issue #92

Comment (dave-connors-3) [reply integration_tests/models/dbt_project_evaluator_schema_tests/graph.yml:77]: @graciegoheen are you suggesting we punt on this for this particular PR, and have a more robust docs section for using/overriding/customizing variables?

File: integration_tests/models/dbt_project_evaluator_schema_tests/graph.yml [hunk 2]
```diff
@@ -72,3 +72,21 @@ models:
         tests:
           - unique
           - not_null
+
+  - name: stg_naming_convention_folders
```
Comment (graciegoheen) [inline integration_tests/models/dbt_project_evaluator_schema_tests/graph.yml:76]: Not sure I follow this description - ` into a model to join into`. Do you mean `This table parses the folder name variables into a model to help define structure tests`?

**Review by graciegoheen — APPROVED** [review]
One small comment, otherwise looks good

**Review by b-per — APPROVED** [review]
🥖

**Pull request merged**